builder.md

```diff
@@ -81,14 +90,16 @@ Each template repository MUST have a templates folder containing a config.yaml a
 These files have the following structure:
 
     default:
-        extension: .file-extension
-        output: output-directory-name
+      extension: .file-extension
+      output: output-directory-name
 
     additional:
-        extension: .file-extension
-        output: output-directory-name
+      styles: [base16]
+      filename: "output-directory-name/{{ scheme-style }}-{{ scheme-slug }}.file-extension"
 
-This example specifies that a Builder is to parse two template files: `templates/default.mustache` and `templates/additional.mustache`. `extension` defines the extension of the file that will be produced by a Builder, e.g. `base16-default-dark.file-extension`, and `output` defines the output directory that will be created within the template repository's root directory where the processed templates will be created, e.g. `output-directory-name/base16-default-dark.file-extension`.
+This example specifies that a Builder is to parse two template files: `templates/default.mustache` and `templates/additional.mustache`. `extension` defines the extension of the file that will be produced by a Builder, e.g. `base16-default-dark.file-extension`, and `output` defines the output directory that will be created within the template repository's root directory where the processed templates will be created, e.g. `output-directory-name/base16-default-dark.file-extension`. The `styles` key defaults to an array containing only `base16` and limits which schemes will be built for these templates.
```

Comment:
Ah, here we go...  the explanation. :-)  Should `[base16]` really be the default when I thought we were moving away from that?  And I'm also not sure we've put enough thought into what having an array here would mean... what if this was:

```
styles: [
  "base16", "base17", "base9"
}
```

What if I want to build both "3024" base16 AND "3024" base9, is that impossible?  Or if we're glob("**") doe they just magically end up in the correct output bin?

Reply:
The idea behind defaulting to `[base16]` was to allow templates to be backwards compatible - they wouldn't need any changes to continue working.

Making it an array was combatting the current format: because it's a mapping of "template file" to "template config", each template file can only be used once. That being said, I'm not sure if it makes sense to support multiple sets of styles with one template? Maybe it makes more sense if we're doing aliases as well?

Reply:
>  was to allow templates to be backwards compatible

Oh you mean if a template doesn't specify then it's `base16`.. ah... ok I agree with that... `system` should work the same way... non specified systems should be `base16`.

> I'm not sure if it makes sense to support multiple sets of styles with one template?

I think it could. Esp in a "they are more or less all the same" world...